### PR TITLE
removed repeated word

### LINKED ISSUE #5128: [Handbook] Word repetition
### Description

Currently the word "other" is repeated twice (after each other) in a sentence. 

Path: Handbook > [General FAQ's](https://layer5.io/community/handbook/faq) > last answer --> "...the community calendar, while other other times they are simply self-organized by interested contributors."
### Expected Behavior

Removing the duplicate word _other,_ would fix the grammar/typo i.e "...the community calendar, while other times they are simply self-organized by interested contributors."

### Screenshots

img
### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Harsha-10**
**Description**

This PR fixes #5128

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 389d2f4deb48f993a67fa6aec9807853f5ea5556 at: https://6555f4eeac7e79708c85a388--layer5.netlify.app

**Review by ankushbhardwaj408 — CHANGES_REQUESTED** [review]

File: src/sections/Community/Handbook/faq.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const Faq = () => {
...
+            <p>A: Yes, outside of the regularly scheduled meetings on the community calendar (meet.layer5.io), there are any number of other meetings that occur to facilitate deeper levels or more concentrated attention to particular bodies of work (e.g. a design spec review). On occasion, these one-off meetings are scheduled in advance and put onto the community calendar, while other times they are simply self-organized by interested contributors. Often contributors will use Slack huddles or Zoom meetings to facilitate the discussion. If you need a new meeting placed onto the community calendar, please reach out to a <Link to="/community/community-managers">Community Manager</Link></p>
```
[the remark this answers is not in the document]
Comment (Harsha-10) [reply src/sections/Community/Handbook/faq.js]: @ankushbhardwaj408  Done

**Comment by l5io** [comment]
🚀 Preview for commit ec1fac7edc96f2a2f4f2beba5beae904af8e848d at: https://655701a27abbc6596b054e10--layer5.netlify.app

**Review by ankushbhardwaj408 — APPROVED** [review]
LGTM!

**Review by Mohith234 — APPROVED** [review]
LGTM, Thanks for the quick fix @Harsha-10 🚀

**Pull request merged**